### ISSUE TITLE
Upgrade to Spring Framework 6.2.0-RC2

### ISSUE THREAD

**Issue by wilkinsona** [spring-projects/spring-boot#42562]

**Comment by wilkinsona** [comment]
Moving to snapshots is currently blocked by https://github.com/spring-projects/spring-framework/issues/33678.

**Comment by wilkinsona** [comment]
@Shivam1-123 please stop spamming the issue tracker with `/assign` comments.

**Comment by Shivam1-123** [comment]
@wilkinsona Is this problem resolved? I just wanted to work on it.

**Comment by snicoll** [comment]
superseded by #42833
